File: benchmark/2026-09-17-7677581(latest)/semantic-compression/_superseded_run1/probes/python/F06.P2.py
def main() -> int:
# BEGIN PROBE F06.P2
    def divmod2(a: int, b: int) -> tuple[int, int]:
        return a // b, a % b

    q, r = divmod2(7, 3)
    return q + r
# END PROBE F06.P2


print(main())

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F06.P2.java
class Main {
    static int caller() {
        // BEGIN PROBE F06.P2
        return switch (divmod2(7, 3)) { case QR(int q, int r) -> q + r; };
    }

    record QR(int q, int r) {}

    static QR divmod2(int a, int b) { return new QR(a / b, a % b); }
        // END PROBE F06.P2

    public static void main(String[] args) {
        System.out.println(caller());
    }
}
